### PR TITLE
Fix -在开启认证的情况下增加token解析认证处理方式

### PR DESCRIPTION
**Pull request by MentosL**
Fix -在开启认证的情况下增加token解析认证处理方式
ref ： #10169 #9109 #8107 #10176

Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/controller/UserController.java [hunk 1]
```diff
@@ -152,7 +152,7 @@ public Object updateUser(@RequestParam String username, @RequestParam String new
                 response.sendError(HttpServletResponse.SC_FORBIDDEN, "authorization failed!");
                 return null;
             }
-        } catch (HttpSessionRequiredException e) {
+        } catch (HttpSessionRequiredException | AccessException e) {
```
Comment (KomachiSion) [inline plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/controller/UserController.java:155]: AccessException should be `authorization failed!`?

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Comment by KomachiSion** [comment]
CI can't pass, please fix them and discuss for review comment.

File: plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/controller/UserController.java (hunk 1 above)
Comment (MentosL) [reply plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/controller/UserController.java:155]: Yes, I'll reorganize it

**Comment by KomachiSion** [comment]
The checkstyle seems no pass, you can rebase the `develop` branch  and run `mvn -Prelease-nacos -DskipTests clean install` in local first

**Comment by MentosL** [comment]
> The checkstyle seems no pass, you can rebase the `develop` branch and run `mvn -Prelease-nacos -DskipTests clean install` in local first

ok

**Review by KomachiSion — DISMISSED** [review]

**Comment by KomachiSion** [comment]
The checkstyle still can pass, Please run `mvn -Prelease-nacos -DskipTests clean install` in local and fix check style first.

**Review by KomachiSion — APPROVED** [review]

**Comment by MentosL** [comment]
@KomachiSion  pls check it and approve it thanks

**Pull request merged**